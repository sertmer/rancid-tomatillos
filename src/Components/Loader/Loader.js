import React from 'react';
import './Loader.scss';

const Loader = () => {
  return (
    <>
      <img src='../../images/film_projector.gif' alt='old timey film projector' />
    </>
  )
}
