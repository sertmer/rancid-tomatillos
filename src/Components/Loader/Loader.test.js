import React from 'react';
import Loader from './Loader';
import { shallow } from 'enzyme';

describe('Loader', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Loader />)
  })

  it.skip('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})
