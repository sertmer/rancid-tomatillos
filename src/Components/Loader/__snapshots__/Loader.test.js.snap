// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should match the snapshot 1`] = `ShallowWrapper {}`;
